src/Main.py

```diff
@@ -0,0 +1,636 @@
+#!/usr/bin/env python3
+import sys
+import os
+import gi
+import subprocess
+import gettext
+gi.require_version("Gtk", "3.0")
+gi.require_version("Handy", "1")
+gi.require_version("Vte", "2.91")
+from threading import Thread
+from gi.repository import Gtk, Handy, Gdk, Gio, GLib, Vte
+
+gettext.install("pardus-boot-repair", "/usr/share/locale/")
+Handy.init()
+
+APP_DIR = os.path.dirname(os.path.realpath(__file__))
+
+resource = Gio.resource_load(APP_DIR + "/data/tr.org.pardus.boot-repair.gresource")
+Gio.Resource._register(resource)
+
+class Application(Gtk.Application):
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+          *args, 
+          application_id="tr.org.pardus.boot-repair",
+          **kwargs
+        )
+
+        # main
+        self.builder = Gtk.Builder()
```

Comment:
headerbar window controls are missing.

Reply:
We can add show logs button on headerbar. 
You can add about dialog.

Pardus footer missing.

Reply:
you must clear log screen before new operations.

Reply:
You can add operating system name in partition selection page.